ga4gh/cli.py

```diff
@@ -735,6 +904,21 @@ def addEffectsArgument(parser):
             """)
 
 
+def addFeatureTypesArgument(parser):
+    parser.add_argument(
+        "--featureTypes", "-n", default="",
```

Comment:
It used to be called `--ontologyTerm` and `-o` was already taken. I'm changing it now.